roles/openshift_logging/templates/elasticsearch.yml.j2

```diff
@@ -7,8 +7,7 @@ script:
 
 index:
   number_of_shards: 1
-  number_of_replicas: 0
-  auto_expand_replicas: 0-2
+  number_of_replicas: 1
```

Comment:
Let's make this `0` by default, and give the customers a setting to change it to the value they want.

I would think both of these numbers (shard and replica counts) should be templated to allow a variable to change their values.